src/program/snabbnfv/nfvconfig.lua

```diff
@@ -25,10 +31,11 @@ function load (file, pciaddr, sockpath)
       local name = port_name(t)
       local NIC = "NIC_"..name
       local Virtio = "Virtio_"..name
-      config.app(c, NIC, Intel82599, {pciaddr = pciaddr,
-                                      vmdq = true,
-                                      macaddr = mac_address,
-                                      vlan = vlan})
+      config.app(c, NIC, require(device_info.driver).driver,
```

Comment:
(Just reminds me: Generally in Snabb Switch we need a better way to refer to apps and network interfaces. Global names for the driver like `apps.intel.intel10g.driver` are excessively verbose, and PCI addresses for network interfaces are excessively specific - won't work for e.g. raw socket on `eth0` even though some applications would work with one.)